app/services/ui-development.html

```diff
@@ -100,6 +100,11 @@ <h2 class="title my-4">Front-end development</h2>
         <p>
           We have extensive experience in working with:
         </p>
+        <picture class="d-flex justify-content-between mt-5">
```

Comment:
Please correct this. One picture tag can have only one image. We add multiple image tags only when we are serving images for different screen sizes.

Ex: 
`<picture>
<img scr="..." alt="...">
</picture>`

Reply:
Picture tag added to all images in service pages.